src/python/WMCore/MicroService/MSPileup/MSPileupData.py

```diff
@@ -145,6 +157,19 @@ def updatePileup(self, doc, rseList=None, validate=True):
         if pname == '':
             pname = doc.get('pileupName', '')
             spec = {'pileupName': pname}
+
+        # add transition record if doc contains new fraction
+        if 'containerFraction' in doc:
```

Comment:
do we need to check if "this containerFraction" is different than "previous containerFraction"? Such that we only update it when there is a real change to the containerFraction.

Reply:
done